cvat/apps/engine/serializers.py

```diff
@@ -156,10 +150,36 @@ class TasksSummarySerializer(_CollectionSummarySerializer):
 class CommentsSummarySerializer(_CollectionSummarySerializer):
     pass
 
+class BasicSummarySerializer(serializers.Serializer):
+    url = serializers.URLField(read_only=True)
+    count = serializers.IntegerField(read_only=True)
 
-class IssuesSummarySerializer(_CollectionSummarySerializer):
-    pass
+    def to_representation(self, instance):
```

Comment:
I don't think you need to override `to_representation` here. You can replace the fields with `SerializerMethodField`s, and that should cause the methods to be invoked automatically.

Reply:
I like the proposal but the method has common logic. It handles the case when the request is None. I will prefer to leave it as is. Otherwise, I have to repeat it 4 times.